packages/calling-component-bindings/src/utils/videoGalleryUtils.ts

```diff
@@ -42,13 +50,15 @@ const memoizedAllConvertRemoteParticipant = memoizeFnAll(
     isMuted: boolean,
     isSpeaking: boolean,
     videoStreams: { [key: number]: RemoteVideoStreamState },
+    state: RemoteParticipantConnectionState,
```

Comment:
Let's call this `connectionState` everywhere. `state` is too generic, and already has at least two distinct meanings in our package.

Reply:
Ugh. I see that this is called `RemoteParticipantState` and the field is called `state` in the headless API.

So... let's stick to `state` to be consistent with that (bad) naming.